src/config/rt.rs

```diff
@@ -164,6 +166,7 @@ impl RtcServe {
         let watch = Arc::new(RtcWatch::new(build_opts, watch_opts, tools, hooks, !opts.no_autoreload)?);
         Ok(Self {
             watch,
+            address: opts.address.unwrap_or("0.0.0.0".to_string()),
```

Comment:
Looks like there is a linter issue on this line (using `unwrap_or_else(|| ...)` to avoid the allocation when it is not necessary). Not sure why the CI lint is not showing up here. I'll have to update it later.